app/helpers/pages_helper.rb

```diff
@@ -33,6 +39,15 @@ def plugin_title(plugin)
     "#{plugin.name}#{detail}"
   end
 
+  def plugin_icon(plugin)
+    registered = {
+      action: 'hand-rock-o',
+      thermometer: 'neuter'
+    }
+    name = plugin.name.underscore.to_sym
+    registered.has_key?(name) ? registered[name] : 'cubes'
```

Comment:
`#fetch` is a standard `Hash` method created for this case. It's not cryptic, and should be as familiar as the ternary operator.